### PR TITLE
update hotel.sql & load-samples.sql

### PR DESCRIPTION
**Pull request by arlcurten**
Modification:
1. added Produces table since M:N notation
2. added an extra attribute "quantity" to Produces table
(we may need to modify ERD later)
3. price of invoice and room were out of range, so made it to (6,2),
4. moved FK defining after data inserted to avoid conflicts when executing script

Others:
1. In order to successfully execute in My SQL, I commented password attribute in guest/employee. Add them back later 
2. Insert data in a large query for each table instead of multi-query, since its easier to execute when debugging

### CODE REVIEW

File: SQL/hotel.sql [hunk 1]
```diff
@@ -51,10 +57,10 @@ CREATE TABLE Employee(
     employeeSSN CHAR(9) NOT NULL, -- PK (reflect this change on ERD)
     employeeFName VARCHAR(20) NOT NULL,
     employeeLName VARCHAR(20) NOT NULL,
-    employeeSex CHAR(1), -- M: men, W:women
+    -- employeeSex CHAR(1), -- M: men, W:women  -- Dave:could we remove the attribute?
```
Comment (swdev1202) [inline SQL/hotel.sql]: Yeah, we don't really care

File: SQL/hotel.sql [hunk 2]
```diff
@@ -51,10 +57,10 @@ CREATE TABLE Employee(
...
     employeeDOB DATE,
     employeeSalary INT,
-    employeePassword VARCHAR(20) NOT NULL, -- depending on  the encryption
+    -- employeePassword VARCHAR(20) NOT NULL, -- depending on  the encryption  -- Dave:let's add it later
```
Comment (swdev1202) [inline SQL/hotel.sql]: Agreed

File: SQL/hotel.sql [hunk 3]
```diff
@@ -7,15 +7,15 @@ CREATE TABLE Guest(
     guestID VARCHAR(20) NOT NULL, -- PK
     guestFName VARCHAR(20),
     guestLName VARCHAR(20),
-    guestPhoneNumber CHAR(10),
-    guestPassword VARCHAR(20) NOT NULL -- VARHCAR size depending on the encrpytion 
+    guestPhoneNumber CHAR(10)
+    -- guestPassword VARCHAR(20) NOT NULL -- VARHCAR size depending on the encrpytion  -- Dave:let's add it later
```
Comment (swdev1202) [inline SQL/hotel.sql]: Agreed

File: SQL/hotel.sql [hunk 4]
```diff
@@ -70,14 +76,19 @@ ALTER TABLE Invoice ADD CONSTRAINT PK_Invoicec_Num PRIMARY KEY(invoiceNum);
 ALTER TABLE Reservation ADD CONSTRAINT PK_Reservation_Num PRIMARY KEY(bookNumber);
 ALTER TABLE Room ADD CONSTRAINT PK_Room_Num PRIMARY KEY(roomNumber);
 ALTER TABLE Orders ADD CONSTRAINT PK_Order_Num PRIMARY KEY(orderNumber);
+ALTER TABLE Produces ADD CONSTRAINT PK_Produce_orNum_fID PRIMARY KEY(orderNumber, foodID);
```
Comment (swdev1202) [inline SQL/hotel.sql]: Do we need to have a composite primary key in M:N relation?

File: SQL/hotel.sql [hunk 5]
```diff
@@ -70,14 +76,19 @@ ALTER TABLE Invoice ADD CONSTRAINT PK_Invoicec_Num PRIMARY KEY(invoiceNum);
...
 
 -- assigning FK to each table --
-ALTER TABLE Invoice ADD CONSTRAINT FK_Invoice_Guest_ID FOREIGN KEY(guestID) REFERENCES Guest(guestID);
-ALTER TABLE Reservation ADD CONSTRAINT FK_Reservation_Guest_ID FOREIGN KEY(guestID) REFERENCES Guest(guestID);
-ALTER TABLE Reservation ADD CONSTRAINT FK_Room_Num FOREIGN KEY(roomNum) REFERENCES Room(roomNumber);
-ALTER TABLE Orders ADD CONSTRAINT FK_Orders_Guest_ID FOREIGN KEY(guestID) REFERENCES Guest(guestID);
-ALTER TABLE Employee ADD CONSTRAINT FK_Employee_Dept_Num FOREIGN KEY(dno) REFERENCES Department(departmentNumber);
-ALTER TABLE Department ADD CONSTRAINT FK_Manager_SSN FOREIGN KEY(manager_SSN) REFERENCES Employee(employeeSSN);
\ No newline at end of file
+/*
```
Comment (swdev1202) [inline SQL/hotel.sql]: Why are they commented out?

File: SQL/hotel.sql (hunk 4 above)
Comment (arlcurten) [reply SQL/hotel.sql]: Yes, that's the rule of creating new table from relationship.

File: SQL/load-samples.sql [hunk 6]
```diff
@@ -1,76 +1,207 @@
...
+(14, '198743987', 'Rosa', 'Herrera', '1981-04-04', 79000, 1),
+(15, '750687067', 'Ana', 'Ross', '1992-03-28', 82000, 0),
+(16, '932465322', 'Alesha', 'Shepherd','1992-01-05', 85000, 0),
+(17, '924321920', 'Connor', 'John', '1988-07-11', 83000, 0),
+(18, '335332264', 'Dennis', 'Thompson', '1989-04-12', 75000, 1),
+(19, '904654673', 'Lauren', 'Chambers', '1985-05-26', 76000, 0),
+(20, '255902532', 'Amelia', 'Lee', '1987-12-08', 75000, 0);
+   -- FrontDest:15, HR:5; total:20 --
+   
+ALTER TABLE Employee ADD CONSTRAINT FK_Employee_Dept_Num FOREIGN KEY(dno) REFERENCES Department(departmentNumber)  ON DELETE CASCADE  ON UPDATE CASCADE;
```
Comment (swdev1202) [inline SQL/load-samples.sql]: aha I found it

**Pull request merged**